hnn_core/viz.py

```diff
@@ -273,6 +273,11 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
     elif average:
         dpl = dpl + [average_dipoles(dpl)]
 
+    for single_dpl in dpl:
+        if not isinstance(single_dpl, Dipole):
+            raise ValueError('dpl should be of type Dipole or list of Dipole, '
+                             f'but {single_dpl} is a {type(single_dpl)}')
```

Comment:
If `single_dpl` happens to be a list or ndarray, this error message will be long and potentially hard to read.

Reply:
What alternative do you suggest moving forward here?

Reply:
```suggestion
            raise ValueError('dpl should be of type Dipole or list of Dipole '
                             f'but is of type {type(single_dpl)}')
```

Reply:
We just don't want the traceback to be huge if the user accidentally enters `dpl` as a `list of list` or `ndarray`.